Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -134,4 +149,21 @@ class EventNotificationsManager {
             self.sendNotifications(forEvents: foundEvents)
         })
     }
+
+    fileprivate func isUnsent(event: Event) -> Bool {
+        guard let placeEvents = sentNotifications[event.placeId] else {
+                return true
+        }
+        return !placeEvents.contains(event.description)
+    }
+
+    fileprivate func markAsSent(event: Event) {
+        var sent = sentNotifications
+        if sent.keys.contains(event.placeId) {
+            sent[event.placeId]?.append(event.description)
+        } else {
+            sent[event.placeId] = [event.description]
```

Comment:
nit: I'm not sure `event.description` is guaranteed not to change – maybe `event.url` is slightly safer? In any case, in the future, it'd be good to create event IDs for our internal use.